pkg/controllers/federatedresourcequota/federated_resource_quota_sync_controller.go

```diff
@@ -190,16 +183,13 @@ func (c *SyncController) buildWorks(quota *policyv1alpha1.FederatedResourceQuota
 		}
 	}
 
-	if len(errs) > 0 {
-		return errors.NewAggregate(errs)
-	}
-	return nil
+	return errors.NewAggregate(errs)
 }
 
 func extractClusterHardResourceList(spec policyv1alpha1.FederatedResourceQuotaSpec, cluster string) corev1.ResourceList {
-	for index := range spec.StaticAssignments {
-		if spec.StaticAssignments[index].ClusterName == cluster {
-			return spec.StaticAssignments[index].Hard
+	for _, staticAssignment := range spec.StaticAssignments {
+		if staticAssignment.ClusterName == cluster {
+			return staticAssignment.Hard
```

Comment:
Why?

Reply:
I don't think so. It's a kind of coding style. I don't think we need to work on that.

Reply:
ok. thanks